src/sonic-config-engine/sonic-cfggen

```diff
@@ -33,6 +33,11 @@ from swsssdk import ConfigDBConnector
 from collections import OrderedDict
 from natsort import natsorted
 
+def sift_up_ipv4(value):
```

Comment:
sort_by_prefix

add comments:
for interface name is the same, then sort by prefix. v4 address is before v6 address